backend/src/openarchiefbeheer/destruction/tasks.py

```diff
@@ -123,5 +123,6 @@ def complete_and_notify(pk: int) -> None:
     destruction_list.save()
 
     destruction_list.set_status(ListStatus.deleted)
+    destruction_list.generate_destruction_report()
```

Comment:
What happens if creating the destruction report fails due to the system being out of disk space,

- can we reconstuct the file later?
- should the destruction list status be changed?

Reply:
An error would be raised, and this would set the destruction list internal status to failed: https://github.com/maykinmedia/open-archiefbeheer/blob/d6feb2bb6175e748ceb28ef4bd6db10a77bbbbb4/backend/src/openarchiefbeheer/destruction/tasks.py#L97
Then, because the metadata is in the items, when the task is retried it would be tried again.